packages/@cldcvr/flow-form-builder/src/components/f-form-builder/f-form-builder.ts

```diff
@@ -95,35 +98,61 @@ export class FFormBuilder extends FRoot {
    */
   build() {
     this.formRef = createRef();
-    return html`<form
+    return html`<f-form
       name="sampleForm"
       @submit=${this.onSubmit}
       @input=${this.handleFormChange}
       ${ref(this.formRef)}
+      size=${ifDefined(this.config.fieldSize)}
+      category=${ifDefined(this.config.category)}
+      variant=${ifDefined(this.config.variant)}
+      ?separator=${this.config.groupSeparator}
+      gap=${ifDefined(this.config.gap)}
     >
-      <f-div direction="column" gap="small">
-        ${Object.entries(this.config.groups).map(([name, gr]) => {
-          const groupWrapperRef: Ref<HTMLElement> = createRef();
-          if (gr.show) {
-            this.state.showFunctions.set(groupWrapperRef, gr.show);
-          }
-          return html`
-            <f-div gap="medium" direction="column" ${ref(groupWrapperRef)}>
-              <f-text variant="heading">${name}</f-text>
-              <f-div
-                gap="medium"
-                .direction=${gr.direction === "horizontal" ? "row" : "column"}
-              >
-                ${this.buildFields(name, gr.fields)}
-              </f-div>
-              <f-divider></f-divider>
-            </f-div>
-          `;
-        })}
-        <br />
-        <slot></slot>
+      <f-div padding="none" gap="x-small" direction="column" width="fill-container">
+        <f-div padding="none" gap="small" direction="row" width="hug-content" height="hug-content">
+          <f-div padding="none" direction="row" width="hug-content" height="hug-content">
+            <f-text variant="heading" size="medium" weight="regular"
+              >${this.config?.label?.title}</f-text
+            >
+          </f-div>
+          ${this.config?.label?.iconTooltip
+            ? html` <f-icon
+                source="i-question-filled"
+                size="small"
+                state="default"
+                clickable
+              ></f-icon>`
+            : ""}
+        </f-div>
+        ${this.config?.label?.description
+          ? html` <f-text variant="para" size="medium" weight="regular"
+              >${this.config?.label?.description}</f-text
+            >`
+          : ""}
       </f-div>
-    </form>`;
+      ${Object.entries(this.config.groups).map(([name, gr]) => {
+        const groupWrapperRef: Ref<HTMLElement> = createRef();
+        if (gr.showWhen) {
+          this.state.showFunctions.set(groupWrapperRef, gr.showWhen);
+        }
+        return html`
+          <f-form-group
+            .direction=${gr.direction}
+            .variant=${gr.variant}
+            .label=${gr.label ?? { title: name }}
+            gap=${ifDefined(gr.gap)}
+            .collapse=${gr.isCollapsible ? "accordion" : "none"}
+            ${ref(groupWrapperRef)}
+          >
+            ${this.buildFields(name, gr.fields, gr)}
+          </f-form-group>
+        `;
+      })}
+      <f-form-group>
+        <slot @click=${this.onSubmit}></slot>
```

Comment:
don't add click-on slot , because we never know if that is submit button or with cancel button.

By default, it submits if the button type is submit or else they handle how to submit.